util/chunk/mutrow_test.go

```diff
@@ -78,6 +78,13 @@ func (s *testChunkSuite) TestMutRow(c *check.C) {
 	row = mutRow.ToRow()
 	c.Assert(row.GetJSON(0), check.DeepEquals, j)
 	c.Assert(row.GetTime(1), check.DeepEquals, t)
+
+	retTypes := []*types.FieldType{types.NewFieldType(mysql.TypeDuration)}
+	chk := New(retTypes, 1, 1)
+	chk.AppendMyDecimal(0, types.NewDecFromFloatForTest(1.123))
```

Comment:
Why we append a decimal to a Duration column?

Reply:
Oh..you're right.. I append the wrong thing...